pkg/comp-functions/functions/vshnnextcloud/deploy.go

```diff
@@ -135,70 +139,19 @@ func DeployNextcloud(ctx context.Context, comp *vshnv1.VSHNNextcloud, svc *runti
 	return nil
 }
 
-func addPostgreSQL(svc *runtime.ServiceRuntime, comp *vshnv1.VSHNNextcloud) error {
-	// Unfortunately k8up and stackgres backups don't match up very well...
-	// if no daily backup is set we just do the default.
-	retention := 6
-	if comp.Spec.Parameters.Backup.Retention.KeepDaily != 0 {
-		retention = comp.Spec.Parameters.Backup.Retention.KeepDaily
```

Comment:
I'm deleting this function as it's unused, we switched logic to "newPostgresqlDependencyBuilder"